gprofiler/profilers/python.py

```diff
@@ -281,7 +291,10 @@ class PythonEbpfProfiler(ProfilerBase):
     _EVENTS_BUFFER_PAGES = 256  # 1mb and needs to be physically contiguous
     # 28mb (each symbol is 224 bytes), but needn't be physicall contiguous so don't care
     _SYMBOLS_MAP_SIZE = 131072
-    _DUMP_SIGNAL = signal.SIGUSR2
```

Comment:
Undo those changes - we don't need the `PythonEbpfProfiler` on Windows, so this class shouldn't even be defined. We can move it to another file or we can put it under `if windows()`. I think another file makes more sense. Let's start with avoiding the import of `java.py`, `php.py` on Windows and then decide what we do with `python.py` which is mixed with some things that we need on Windows, and some that we don't.

Reply:
The PythonProfiler class in gprofiler.profilers.python imports PythonEbpfProfiler and has _create_ebpf_profiler defined as a member function. What I have done now is to move the PythonEbpfProfiler and PythonEbpfError to its own file and import that module into the python.py file. I also still have to use the is_windows helper function to switch between signal.SIGUSR2 and signal.SIGFPE as signal.SIGUSR2 is not supported on Windows.

Reply:
It's good to move to a separate file. It doesn't make sense though to change the signal because PyPerf requires SIGUSR2 (and PyPerf itself is a Linux only application). So we just need to make sure that this code (i.e the definition of `_DUMP_SIGNAL`) doesn't run on Windows :)

Reply:
I included a check of is_windows in the definition of _DUMP_SIGNAL. PythonEbfProfiler is used by the class PythonProfiler. So, I can't choose to only import PythonEbfProfiler on Linux - Windows also requires it.

Reply:
We've since completed this split. Resolving this